v21.2/deploy-app-vercel.md

```diff
@@ -9,6 +9,11 @@ docs_area: get_started
 
 This tutorial shows you how to use [Vercel](https://vercel.com/) to deploy a web application built with [Next.js](https://nextjs.org/), [Express](https://expressjs.com/), [Prisma](https://www.prisma.io/), and {{ site.data.products.serverless }}.
 
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="browser"><strong>Use your browser</strong></button>
+    <button class="filter-button page-level" data-scope="local"><strong>Use the command line</strong></button>
+</div>
+
 ## Prerequisites
 
 Before starting the tutorial, do the following:
```

Comment:
> Re: deploy button, I thought we were using this project that doesn't have a deploy button: https://github.com/cockroachdb/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes. Do you want to go with the javascript example instead?

Looks like I got confused between multiple projects with nearly the same name. I meant to use this project with the deploy button: https://github.com/cockroachdb/prisma-examples/tree/latest/javascript/rest-nextjs

After thinking about it, I don't think it's interesting to document the deploy button flow because it's already pretty easy. So we should document the flow as if the deploy button is not available, which is what you've done. I'll take another pass at the review with that in mind.

Reply:
I'm a fan of keeping this for the referral ID.